Libraries/PeriphDrivers/Include/MAX32690/mxc_sys.h

```diff
@@ -430,14 +425,6 @@ void MXC_SYS_RISCVRun(void);
  */
 void MXC_SYS_RISCVShutdown(void);
 
-/**
- * @brief Set the clock source for the RISC-V core.
- * 
- * @param clock The clock source to set
- * @returns 0 if successful, @ref MXC_Error_Codes on errors
- */
-int MXC_SYS_RISCVClockSelect(mxc_sys_riscv_clock_t clock);
```

Comment:
Leave this in, but add a note in the comments and a deprecated attribute to notify users and prevent breaking existing projects using this function.

Something like this:
```
inline __attribute__((deprecated("09-04-2024: Use the MXC_LP_RISCVClockSelect function, and the mxc_lp_riscv_clock_t enum instead.  See lp.h for more details."))) int MXC_SYS_RISCVClockSelect(mxc_sys_riscv_clock_t clock)
{
    MXC_LP_RISCVClockSelect((mxc_lp_riscv_clock_t)clock);
}
```

Include lp.h to prevent implicit declaration warnings.

Reply:
Hi - totally agree generally about backwards compatibility.
However, this was only added very recently (a few weeks ago) and it's highly unlikely anyone is using it yet. I hoped to be able to fix it without having to leave the clutter in the sources.
Still, happy to change to use deprecated if you'd prefer not to take the risk.

Reply:
Oh, as long as this wasn't in the latest official release (Last release was ~Feb 2024), don't worry about it.